packages/preferences/src/browser/util/preference-types.ts

```diff
@@ -134,6 +134,24 @@ export namespace PreferencesCommands {
         category: 'Preferences',
         label: 'Open Workspace Preferences',
     };
+
+    export const OPEN_USER_PREFERENCES_JSON: Command = {
+        id: 'workbench.action.openSettingsJson',
+        category: 'Preferences',
+        label: 'Open Preferences (JSON)'
+    };
+
+    export const OPEN_WORKSPACE_PREFERENCES_JSON: Command = {
+        id: 'workbench.action.openWorkspaceSettingsFile',
+        category: 'Preferences',
+        label: 'Open Workspace Preferences (JSON)',
+    };
+
+    export const OPEN_FOLDER_PREFERENCES_JSON: Command = {
+        id: 'workbench.action.openFolderSettingsFile',
+        category: 'Preferences',
+        label: 'Open Folder Preferences (JSON)',
+    };
```

Comment:
As far as I can see there's no normal `Open Folder Preferences` command in Theia right now. Perhaps it is out of scope for this PR, but do you mind adding it? The appropriate frontend UI in the preferences view is already supported for that:

![image](https://user-images.githubusercontent.com/4377073/128162573-da39e167-f8bb-4985-90cf-92aa2d750adb.png)